internal/target/pulp_target.go

```diff
@@ -0,0 +1,33 @@
+package target
+
+const TargetNamePulpOSTree TargetName = "org.osbuild.pulp.ostree"
+
+type PulpOSTreeTargetOptions struct {
+	// ServerAddress for the pulp instance
+	ServerAddress string `json:"server_address"`
```

Comment:
I have the same comment as with credentials... The address can be set also in the worker config, thus it could be marked as `omitempty` here, no?

Reply:
Done

Reply:
Looks like this one is actually not done :sweat_smile: